test/Elastic.Apm.AspNetCore.Tests/MicrosoftExtensionsConfigTests.cs

```diff
@@ -126,7 +130,8 @@ public void ReadConfingsFromEnvVarsViaIConfig()
 		public void LoggerNotNull()
 		{
 			var testLogger = new TestLogger();
-			var config = new MicrosoftExtensionsConfig(GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), testLogger);
+			var config = new MicrosoftExtensionsConfig(GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), testLogger,
+				"test");
```

Comment:
Please see my comment above regarding adding an `internal` constructor overload to `MicrosoftExtensionsConfig`.